.github/workflows/build.yaml

```diff
@@ -42,7 +40,20 @@
       uses: ros-tooling/action-ros-ci@v0.3
       with:
         package-name: |
+          ${{ matrix.BUILD_TYPE == 'source' && env.RMW_MIDDLEWARE_TEST_PACKAGES || '' }}
           rmw_zenoh_cpp
           zenoh_c_vendor
         target-ros2-distro: ${{ matrix.ROS_DISTRO }}
         vcs-repo-file-url: ${{ matrix.BUILD_TYPE == 'source' && env.ROS2_REPOS_FILE_URL || '' }}
+        skip-tests: true
```

Comment:
**Suggestion:** The `skip-tests` flag is set to `true` in the `action-ros-ci` step, which might be unintentional as it prevents any tests from running. If this is not the desired behavior, consider setting this flag to `false`. [possible issue, importance: 9]
```suggestion
        skip-tests: false
```